system/services/admin/DataManagerService.cfc

```diff
@@ -787,7 +797,11 @@ component {
 				}
 
 				filter   &= " )";
-				termDelim = " and ";
+				if ( arrayLen( objEnumFields ) ) {
+					termDelim = " or ";
+				} else {
+					termDelim = " and ";
+				}
```

Comment:
This doesn't really work: it completely changes the way multiple word searches work.

Currently, if your search term is "seb duggan", the "and" term delimiter means that it matches records with BOTH "seb" AND "duggan". But this logic changes it so, if there are enums in the object, it will match EITHER "seb" OR "duggan". So, the termDelim should ALWAYS be "and".

The logic should move up inside the parsedFields loop (after line 795), so if the current field in the loop is an enum, it looks to see if the search term is an enum label _for that enum_, and if it is then we add in something like:

`filter &= " or " & field & " = :#enumParamName#";`

(we'd also need to create a separate index of enum params which would get added to params later in the method).